primitives/runtime/src/traits.rs

```diff
@@ -164,6 +164,57 @@ where
 	}
 }
 
+/// A signature that supports background verification.
+pub trait BackgroundVerify: Verify {
```

Comment:
Are you sure you want to use the term Batch instead of Background?

Batch is a cryptographic term that we not really use.
Here the signatures are not checked in a batch but in a "background task", we had discussed it here: https://github.com/paritytech/substrate/issues/6618